recipes/gplearn/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "gplearn" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/trevorstephens/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: b4ef6139bb8407dd982310a2d227ef11af7a298d1ede82aeb2f0dd55ac3a7bf8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
It does not look like this package builds any C-extensions. If it doesn't please remove this `build` section. If it does, then it can't be `noarch: python`

Reply:
Sure thing, I was wondering about that.